tests/bluetooth/ctrl_llcp/src/main.c

```diff
@@ -464,7 +464,9 @@ void lt_tx(helper_pdu_opcode_t opcode, struct ull_cp_conn *conn, void *param)
 	zassert_equal(0, ret, "k_mem_slab_alloc failed\n");
 
 	pdu = (struct pdu_data *) rx->pdu;
-	helper_pdu_encode[opcode](pdu, param);
+	if (helper_pdu_encode[opcode]) {
```

Comment:
Should we make this into a zassert? If we come here we want to send a PDU, so we should fill in data in this PDU

Reply:
I can do that